### PR TITLE
Changed test_browser_actions to run on FireFox

### PR DESCRIPTION
**Pull request by Rantzur1992**
Signed-off-by: rantz <ran.tzur@testproject.io>

### CODE REVIEW

File: tests/test_browser_actions.robot [hunk 1]
```diff
@@ -42,7 +42,7 @@ Random Actions
 *** Keywords ***
 Init
     ${options_chrome}=     Headless Chrome
```
Comment (vitalybu) [inline tests/test_browser_actions.robot:44]: Headless Chrome? But you initialize the session for Firefox...

File: tests/test_browser_actions.robot [hunk 2]
```diff
@@ -42,7 +42,7 @@ Random Actions
 *** Keywords ***
```
Comment (vitalybu) [inline tests/test_browser_actions.robot:42]: Please build the commit title in the following format:
`module: short description`

For example, "tests: change browsers_actions test to use firefox"

File: tests/test_browser_actions.robot (hunk 1 above)
Comment (Rantzur1992) [reply tests/test_browser_actions.robot:44]: I'm not passing it to the driver anyway, this is there in case.

Comment (Rantzur1992) [reply tests/test_browser_actions.robot]: Removed it instead

File: tests/test_browser_actions.robot (hunk 2 above)
Comment (Rantzur1992) [reply tests/test_browser_actions.robot:42]: Noted, changed.

**Review by vitalybu — APPROVED** [review]

**Pull request merged**